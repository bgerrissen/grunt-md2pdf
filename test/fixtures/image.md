![cookie monster](img/cm.jpg)
![cookie monster small](img/cms.jpg)