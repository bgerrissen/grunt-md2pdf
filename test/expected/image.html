<html><head><style></style></head><body><p><img src="test\fixtures\img\cm.jpg" alt="cookie monster" title="null" />
<img src="test\fixtures\img\cms.jpg" alt="cookie monster small" title="null" /></p>
</body></html>